vipps-in-store-howitworks.md

```diff
@@ -1,9 +1,8 @@
 # Vipps In Store: How It Works
 
-This page shows an example of how you can offer contactless payment to your customers by integrating Vipps in your POS system.
+Vipps i store is designed for stores that want to let customers pay easily and quickly from their own phone, without the need to touch the terminal. Vipps is fully integrated with the cash register system, and thus also with accounting and settlement. Some of our partners now also support mobile checkouts where you can charge directly on the floor in the store.
 
```

Comment:
Won't it be wrong to say `is fully integrated`when this is a guid for how they can integrate? Whoever reads this that want to make an integration, so they would need to do this integration themselves, including the integrations for settlement, and we don't really have anything seperate for accounting and settlement, we only have  https://github.com/vippsas/vipps-developers/tree/master/settlements.

If you still include that this is an example where the example integrator have all this setup, it would probably make more sense?

Reply:
Or maybe mention that Vipps support integrating with settlements and cash registry?

Reply:
I was translating from https://vipps.no/produkter-og-tjenester/bedrift/ta-betalt-i-butikk/vipps-i-kassa/, where it said, "Vipps er helintegrert med kassasystemet, og dermed også med regnskap og oppgjør." But maybe that wasn't quite right.

Reply:
Maybe I can add back the sentence about it being an example, and still keep the first sentence for a bit of introduction...
"Vipps i store is designed for stores that want to let customers pay easily and quickly from their own phone, without the need to touch the terminal. This page shows an example of how you can offer contactless payment to your customers by integrating Vipps in your POS system."